pom.xml

```diff
@@ -2507,5 +2507,26 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>quick-build</id>
+            <activation>
+                <property>
+                    <name>quickly</name>
+                </property>
+            </activation>
+            <properties>
+                <license.skip>true</license.skip>
+                <skipTests>true</skipTests>
+                <enforcer.skip>true</enforcer.skip>
+                <checkstyle.skip>true</checkstyle.skip>
+                <assembly.skipAssembly>false</assembly.skipAssembly>
```

Comment:
Forget about `license.skipDownloadLicenses`, it fails on a full build since the license file is required to allow the `galleon-maven-plugin` to do the provisioning

Reply:
So now, the original configuration runs on my machine `mvn -Dquickly` in 1:40 min and the new one in 1:20. `mvn -Dquickly -Dmaven.repo.local=/tmp/localWfcoreQuickly` also looks good without any failures.

Reply:
Thank @khermano  for pointing out the related WildFly PR.

> I tried mvn -Dquickly -Dmaven.repo.local=/tmp/localWfcore it looked okay, with no failures there, so I think in this case should be fine and I will use it, what do you think?

Well, if we want to use the quickly profile to get builds ready to be tested, then we should switch the strategy and remove `maven.test.skip` and also `assembly.skipAssembly` since the Zip files generated will be used as artifacts for the WildFly integration test suite.

I'm sorry for the back-and-forth, but yes, now I think we should be consistent with the WildFly full quickly profile and follow the same approach allowing using this profile for building and testing WildFly Core. If you don't need the build for testing, you can manually `assembly.skipAssembly` and `maven.test.skip`. 

Would you mind removing both to keep the consistency with WildFly full?

Reply:
Not at all, 
I will remove it right away. I understand that it should be consistent as much as possible.
Thank you very much for your help.